semantic-source/semantic-source.cabal

```diff
@@ -48,7 +48,7 @@ library
     Source.Span
   build-depends:
       aeson          ^>= 1.4.2.0
-    , base            >= 4.13 && < 5
+    , base            >= 4.12 && < 5
```

Comment:
We support 8.6.5 for this package (cf the `tested-with` field, above), and don’t require anything that would require a newer `ghc` anyway.